scale/gcloud_update.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+import subprocess
+from settings import GCLOUD_INSTANCE_GROUP, GCE_ZONE
+from utils import getPods, getPodType, getPodNamespace,\
+    get_singleuser_image_value
+
+import logging
+
+
+def shutdownSpecifiedNode(name):
+    """Deletes the specified node by calling the Google Cloud Engine"""
+    cmd = ['gcloud', 'compute', 'instance-groups', 'managed', 'delete-instances',
+           GCLOUD_INSTANCE_GROUP, '--instances=' + name]
+    p = subprocess.Popen(' '.join(cmd), stdout=subprocess.PIPE, shell=True)
```

Comment:
Note - there are plenty of other possible exploits that use shell=True too :D So it's always a bad idea to use them.